main.js

```diff
@@ -0,0 +1,83 @@
+//bai1
+const getNumber = n => {
+    n == Number
+    for (i = 1; i <= 10; i++) {
+        console.log('${n} * ${i} =', i)
+    }
+}
+
+//bai2
+const takeEvenNumber = n => {
+    if (n >= 1 && n <= 30) {
+        for(let i =2; i <= n; i += 2)
+        console.log(i)
```

Comment:
1. Nhìn kỹ cách viết dấu cách
2. Lệnh con phải tab vào 1 tab so với lệnh cha, 1 tab = 2 spaces. Mỗi 1 lần tab thừa/thiếu 1 space tính là 1 comment
```suggestion
        for(let i = 2; i <= n; i += 2) {
          console.log(i)
        }
```